Core/Helper.php

```diff
@@ -41,4 +53,59 @@ public static function isWirecardPaymentMethod($sPaymentId): bool
     {
         return strpos($sPaymentId, "wd") === 0;
     }
+
+    /**
+     * Returns a list of available payments.
+     *
+     * @return array
+     */
+    public static function getPayments(): array
+    {
+        $oPaymentList = oxNew(ListModel::class);
+        $oPaymentList->init(Payment::class);
+
+        return $oPaymentList->getList()->getArray();
+    }
+
+    /**
+     * Returns a list of available Wirecard payments.
+     *
+     * @return array
+     */
+    public static function getWirecardPayments(): array
+    {
+        return array_filter(self::getPayments(), function ($oPayment) {
+            return $oPayment->oxpayments__wdoxidee_iswirecard->value;
+        });
+    }
+
+    /**
+     * Converts a single level array to multiple levels by splitting keys by a delimiter.
+     *
+     * @param array  $array
+     * @param string $delimiter
+     * @return array
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+     */
+    public static function unflattenArray(array $array, string $delimiter = '.'): array
```

Comment:
The complexity is to high, so the certification of the plugin will fail. Better to refactor it straight away. Where is the function called and needed?